tests/dualtor/test_tor_ecn.py

```diff
@@ -20,6 +20,8 @@
 from tests.common.dualtor.dual_tor_mock import *
 from tests.common.dualtor.dual_tor_utils import get_t1_ptf_ports
 from tests.common.dualtor.dual_tor_utils import rand_selected_interface
+from tests.common.dualtor.dual_tor_utils import get_ptf_server_intf_index
```

Comment:
Looks like this is unused, can it be removed?

Reply:
Done